automation/vars/uni04delta.yaml

```diff
@@ -0,0 +1,80 @@
+---
+vas:
+  uni04delta:
+    stages:
+      - path: examples/dt/uni04delta/control-plane/nncp
+        wait_conditions:
+          - >-
+            oc -n openstack wait nncp
+            -l osp/nncm-config-type=standard
+            --for jsonpath='{.status.conditions[0].reason}'=SuccessfullyConfigured
+        values:
+          - name: network-values
+            src_file: values.yaml
+        build_output: nncp.yaml
+
+      - path: examples/dt/uni04delta/control-plane
+        wait_conditions:
+          - >-
+            oc -n openstack wait osctlplane controlplane
+            --for condition=Ready
+            --timeout=600s
+        values:
+          - name: network-values
+            src_file: nncp/values.yaml
+          - name: service-values.yaml
+            src_file: service-values.yaml
+        build_output: control-plane.yaml
+
+      - path: examples/dt/uni04delta/edpm-pre-ceph/nodeset
+        wait_conditions:
+          - >-
+            oc -n openstack wait
+            osdpns ceph-nodes --for condition=SetupReady
+            --timeout=600s
+        values:
+          - name: edpm-nodeset-values
+            src_file: values.yaml
+        build_output: nodeset-pre-ceph.yaml
+
+      - path: examples/dt/uni04delta/edpm-pre-ceph
+        wait_conditions:
+          - >-
+            oc -n openstack wait
+            osdpd edpm-deployment-pre-ceph --for condition=Ready
+            --timeout=1500s
+        values:
+          - name: edpm-deployment-values
+            src_file: values.yaml
+        build_output: deployment-pre-ceph.yaml
+        post_stage_run:
+          - name: Deploy Ceph
+            type: playbook
+            source: "../../playbooks/ceph.yml"
+            inventory: "${HOME}/ci-framework-data/artifacts/zuul_inventory.yml"
+
+      - path: examples/dt/uni04delta
+        wait_conditions:
+          - >-
+            oc -n openstack wait osdpns
+            openstack-edpm
+            --for condition=SetupReady
+            --timeout=2400s
```

Comment:
can lower the timeout here to for nodeset, as that doesn't need much time, likely leftover from dataplane split.

Reply:
done